src/main/kotlin/pl/edu/agh/gem/external/controller/ExpenseController.kt

```diff
@@ -52,6 +56,16 @@ class ExpenseController(
         return ExpenseResponse.fromExpense(expenseService.getExpense(expenseId, groupId))
     }
 
+    @GetMapping(produces = [APPLICATION_JSON_INTERNAL_VER_1])
+    @ResponseStatus(OK)
+    fun getGroupExpenses(
+        @GemUserId userId: String,
+        @RequestParam groupId: String,
+    ): GroupExpensesResponse {
+        userId.checkIfUserHaveAccess(expenseService.getGroupMembers(groupId))
```

Comment:
now i see one more case, propably to fix in other places what about case when group dont exisits ?